.github/workflows/model-run.yml

```
@@ -0,0 +1,59 @@
+name : Sipnet Model Run
+on :
+  pull_request:
+
+  # allow manual triggering
+  workflow_dispatch:
+
+  schedule:
+    # run Thursday 4:30 AM UTC
+  - cron: '30 4 * * 4'    
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    
+    container: 
+      image: pecan/base:develop
+
+    steps:
+    - name: Checkout source code
+      uses: actions/checkout@v4
+
+    - name: Set up Docker Compose
+      run: |
+          sudo apt-get update
+          sudo apt-get -y install docker-compose
+
+    - name: Build Containers
+      run: |
+        cp docker/env.example .env
+        echo "COMPOSE_PROJECT_NAME=pecan" >> .env
+        echo "PECAN_VERSION=develop" >> .env
+        echo "UID=$(id -u)" >> .env
+        echo "GID=$(id -g)" >> .env
+        docker-compose up -d postgres
+        docker run --rm --network pecan_pecan pecan/db
```

For the next person (me in a year 😄) who wonders "Wouldn't it make sense to write this as a separate step, like...?"
```
- name: db setup
   uses: docker://pecan/db:ci
```
...It might be worth leaving a comment explaining that no, when you run it in a separate step the `--network` option [isn't supported](https://docs.github.com/en/actions/using-jobs/running-jobs-in-a-container#setting-container-resource-options)

---

This reminds me that this line should include an explicit tag too -- the `db:ci` image is currently a little smaller than `db:latest`, but will soon be _much_ smaller (we're working on updating it to contain only the records we need for testing)
```suggestion
        docker run --rm --network pecan_pecan pecan/db:ci
```

---

Also a note of caution for anyone testing this file: this step runs in the same Docker network as the desktop deployment, and WILL DELETE any existing pecan DB it finds there. On GitHub Actions you're starting from a clean VM, but if running locally with e.g. `act -W .github/workflows/model-run.yml -j test`, be prepared to lose your local DB contents.